types/config/utils.go

```diff
@@ -16,6 +16,11 @@ func GetConfigFilePath() string {
 	return path.Join(HomePath, "config.yaml")
 }
 
+// GetGenesisFilePath returns the path to the genesis file
+func GetGenesisFilePath() string {
```

Comment:
I don't see how this is really necessary and, instead, we can't use the config to get the path of the genesis file to be used. It should be enough to use `GenesisFilePath` field of the `parser.Config` field instead. Am I missing something?

Reply:
that's true, I'm gonna cancel this PR and modify forbole/bdjuno#257